lib/browser.ts

```diff
@@ -184,7 +185,7 @@ export class Protractor {
    *
    * @type {Plugins} Object containing plugin funtions from config.
    */
-  private plugins_: Plugins;
+  plugins_: Plugins;
```

Comment:
Maybe double check with Julie, but if they're not actually private then we should just drop the underscore at the end and remove the @private annotations in the jsdoc as well.